src/vcf/main.cpp

```diff
@@ -58,34 +62,122 @@ namespace
             return 1;
```

Comment:
here this returns 1 if level is "stop": three lines above, it checks for "break"